.dotnet.azure/sdk/openai/Azure.AI.OpenAI/src/Custom/FineTuning/AzureCreateJobOperation.Protocol.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Diagnostics.CodeAnalysis;
+
+#nullable enable
+
+namespace Azure.AI.OpenAI.FineTuning;
+
+/// <summary>
+/// A long-running operation for creating a new model from a given dataset.
+/// </summary>
+public class AzureCreateJobOperation : CreateJobOperation

```

Comment:
Does this need to be public?

Reply:
This one does as the delete job is not on the base operation so there is functionality only available to azure

Reply:
Ah, interesting!  Sounds good.  We should go through each public LRO type and validate that there is an answer like this for each one -- if there is not, we likely want to make them internal.  We can always add them to the public API later if needed.

Reply:
I'll make the other ones internal then